src/main/java/io/github/triglav_dataflow/jenkins/trigger/polling_triglav/TimeUnit.java

```diff
@@ -0,0 +1,16 @@
+package io.github.triglav_dataflow.jenkins.trigger.polling_triglav;
```

Comment:
Reformat this file.

Reply:
Fixed. I will take a look to checkstyle rules and see if we can add a warning for that